### PR TITLE
*: add affected rows metrics

### LINKED ISSUE #36435: Show the number of rows inserted/updated/deleted/replaced per second in metrics
## Enhancement

It's better to show the number of rows inserted/updated/deleted/replaced per second in metrics.

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Jack Yu <jackysp@gmail.com>

### What problem does this PR solve?

Issue Number: close #36435

Problem Summary: 

### What is changed and how it works?

Add metrics of affected rows.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

1. start a cluster
2. run sysbench prepare data
3. check Grafana -> TiDB -> Server -> Affected Rows by Type
image
4. compare with QPS panel, this new panel could show the real number of written rows.
image

- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by bb7133 — APPROVED** [review]
LGTM

**Comment by bb7133** [comment]
Please add some scripts or steps:

> 'Manual test (add detailed scripts or steps below)'

**Comment by bb7133** [comment]
PTAL @hawkingrei @crazycs520

**Review by xhebox — COMMENTED** [review]
LGTM, but why are so many changes in grafana?

**Comment by jackysp** [comment]
> LGTM, but why are so many changes in grafana?

I think it may depend on the Grafana being upgraded again on master, so some default exported fields are changed.

File: server/conn.go [hunk 1]
```diff
@@ -1963,8 +1973,8 @@ func (cc *clientConn) prefetchPointPlanKeys(ctx context.Context, stmts []ast.Stm
 		}
 	}
 	pointPlans := make([]plannercore.Plan, len(stmts))
-	var idxKeys []kv.Key //nolint: prealloc
-	var rowKeys []kv.Key //nolint: prealloc
+	var idxKeys []kv.Key // nolint: prealloc
+	var rowKeys []kv.Key // nolint: prealloc
```
Comment (xhebox) [inline server/conn.go:1977]: delete space here, it will cause linter error.

**Review by xhebox — CHANGES_REQUESTED** [review]

File: server/conn.go (hunk 1 above)
Comment (jackysp) [reply server/conn.go]: done

**Review by xhebox — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**